editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5732,11 +5732,9 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 		Ref<Texture2D> texture = Ref<Texture2D>(Object::cast_to<Texture2D>(*res));
 		Ref<PackedScene> scene = Ref<PackedScene>(Object::cast_to<PackedScene>(*res));
 		if (texture != nullptr || scene != nullptr) {
-			bool root_node_selected = EditorNode::get_singleton()->get_editor_selection()->is_selected(EditorNode::get_singleton()->get_edited_scene());
-			String desc = TTR("Drag and drop to add as child of selected node.") + "\n" + TTR("Hold Alt when dropping to add as child of root node.");
-			if (!root_node_selected) {
-				desc += "\n" + TTR("Hold Shift when dropping to add as sibling of selected node.");
-			}
+			String desc = TTR("Drag and drop to add as sibling of selected node (except when root is selected).") +
+					"\n" + TTR("Hold Shift when dropping to add as child of selected node.") +
+					"\n" + TTR("Hold Alt when dropping to add as child of root node.");
 			if (texture != nullptr) {
```

Comment:
```suggestion
					"\n" + TTR("Hold Alt when dropping to add as child of root node.");

			if (texture != nullptr) {
```

Reply:
Do I need to press `Commit suggestion` or did you integrate this change already?

Reply:
You need to apply the changes, best done by adding them manually and using `git commit --amend`, otherwise you'll have to squash your commits

Reply:
looks like that to add this empty line make the formatting checks fail?